### DISCUSSION TITLE
October 2022 experience report

### DISCUSSION THREAD

**Discussion by pbiggar** [darklang/dark#4523] [Feedback]
I wish it were easier to add entries to a datastore.

Creating a dict of a particular shape is annoying, no autofill, no clue

Weird that different DB functions have different signatures (in query, the DB comes first, for good reason).

No Date, only DateTime (I want a date of birth).

The `x` in Password::hash should be closer to the hash, not the next word.

Typing feels weird - it's like I'm filling in a form. If when I pressed <space> my cursor moved (even if there were no space available for it, that might be nicer.

**Comment by pbiggar** [comment]
DB::queryOneWithKey is hard to get the key

**Comment by pbiggar** [comment]
Could do with a larger click surface for clicking "play"

**Comment by StachuDotNet** [comment]
> I wish it were easier to add entries to a datastore.

Same! I imagine this as a Fluid-powered spreadsheet-like interface.
